From 18842c880e5ebd48dd241098a7b53651ad417e42 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Thu, 29 Apr 2021 12:13:11 -0700
Subject: [PATCH] Update github repo urls for gitian and building (#161)

Rebase provenance: this historical change was adapted onto the pinned
Bitcoin ABC base. Source diff is the recorded Lotus V1 replay commit below.

Historical-source-commit: ffdf08f7217162e4e55f00a8c58ec5426b232bb5
Recorded-replay-commit: 78798cce35ec0f8ac7b323eec180076b16ccc074

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0fb513579e..283fd7ad75 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,7 +24,7 @@ add_custom_target(print-project-name
 
 # Package information
 set(PACKAGE_NAME "Bitcoin ABC")
-set(PACKAGE_BUGREPORT "https://github.com/Bitcoin-ABC/bitcoin-abc/issues")
+set(PACKAGE_BUGREPORT "https://github.com/LogosFoundation/lotusd/issues")
 
 # Copyright
 set(COPYRIGHT_YEAR 2026)
diff --git a/contrib/aur/bitcoin-abc-qt/PKGBUILD b/contrib/aur/bitcoin-abc-qt/PKGBUILD
index 3bcc7e68eb..79f67951ca 100644
--- a/contrib/aur/bitcoin-abc-qt/PKGBUILD
+++ b/contrib/aur/bitcoin-abc-qt/PKGBUILD
@@ -9,7 +9,7 @@ url="https://bitcoinabc.org"
 depends=('boost-libs' 'libevent' 'desktop-file-utils' 'qt5-base' 'protobuf' 'openssl' 'zeromq' 'qrencode' 'jemalloc')
 makedepends=('cmake' 'ninja' 'boost' 'qt5-tools' 'python' 'help2man' 'xorg-server-xvfb')
 license=('MIT')
-source=(https://github.com/Bitcoin-ABC/bitcoin-abc/archive/v$pkgver.tar.gz
+source=(https://github.com/LogosFoundation/lotusd/archive/v$pkgver.tar.gz
         bitcoin.conf
         bitcoin.logrotate
         bitcoin.service
diff --git a/contrib/aur/bitcoin-abc/PKGBUILD b/contrib/aur/bitcoin-abc/PKGBUILD
index 86e7da845d..f9eab63783 100644
--- a/contrib/aur/bitcoin-abc/PKGBUILD
+++ b/contrib/aur/bitcoin-abc/PKGBUILD
@@ -9,7 +9,7 @@ url="https://bitcoinabc.org"
 depends=('boost-libs' 'libevent' 'openssl' 'zeromq' 'jemalloc')
 makedepends=('cmake' 'ninja' 'boost' 'python' 'help2man')
 license=('MIT')
-source=(https://github.com/Bitcoin-ABC/bitcoin-abc/archive/v$pkgver.tar.gz
+source=(https://github.com/LogosFoundation/lotusd/archive/v$pkgver.tar.gz
         bitcoin.conf
         bitcoin.logrotate
         bitcoin.service
diff --git a/contrib/debian/copyright b/contrib/debian/copyright
index 70155fa2c6..49cf1d51d6 100644
--- a/contrib/debian/copyright
+++ b/contrib/debian/copyright
@@ -1,7 +1,7 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: Bitcoin ABC
 Upstream-Contact: helpdesk@bitcoinabc.org
-Source: https://github.com/Bitcoin-ABC/bitcoin-abc
+Source: https://github.com/LogosFoundation/lotusd
 
 Files: *
 Copyright: 2009-2017, The Bitcoin Core developers, 2017-2019 The Bitcoin developers
diff --git a/contrib/debian/watch b/contrib/debian/watch
index a2d95a0422..f8b3bbe793 100644
--- a/contrib/debian/watch
+++ b/contrib/debian/watch
@@ -1,3 +1,3 @@
 version=4
 opts=filenamemangle=s/.+\/v?(\d\S+)\.tar\.gz/bitcoin-abc-$1\.tar\.gz/ \
-  https://github.com/Bitcoin-ABC/bitcoin-abc/releases .*/v([\d.]+)\.tar\.gz
+  https://github.com/LogosFoundation/lotusd/releases .*/v([\d.]+)\.tar\.gz
diff --git a/contrib/release/github-release.sh b/contrib/release/github-release.sh
index 8afea29b8d..a7408fef59 100755
--- a/contrib/release/github-release.sh
+++ b/contrib/release/github-release.sh
@@ -174,4 +174,4 @@ else
 fi
 
 echo "Done."
-echo "https://github.com/Bitcoin-ABC/bitcoin-abc/releases/tag/${TAG}"
+echo "https://github.com/LogosFoundation/lotusd/releases/tag/${TAG}"
diff --git a/doc/build-osx.md b/doc/build-osx.md
index 270c321f8e..e4cfe83750 100644
--- a/doc/build-osx.md
+++ b/doc/build-osx.md
@@ -33,7 +33,7 @@ Before you start building, please make sure that your compiler supports C++20.
 
 1. Clone the Bitcoin ABC source code and cd into `bitcoin-abc`
 
-        git clone https://github.com/Bitcoin-ABC/bitcoin-abc.git
+        git clone https://github.com/LogosFoundation/lotusd.git
         cd bitcoin-abc
 
 2.  Build Bitcoin ABC:
diff --git a/doc/build-unix.md b/doc/build-unix.md
index 67ac5ea018..fd673589ab 100644
--- a/doc/build-unix.md
+++ b/doc/build-unix.md
@@ -235,7 +235,7 @@ This example lists the steps necessary to setup and build a command line only,
 non-wallet distribution of the latest changes on Arch Linux:
 
     pacman -S base-devel boost cmake git libevent ninja python
-    git clone https://github.com/Bitcoin-ABC/bitcoin-abc.git
+    git clone https://github.com/LogosFoundation/lotusd.git
     cd bitcoin-abc/
     mkdir build
     cd build
diff --git a/doc/build-windows.md b/doc/build-windows.md
index 03708523d6..663e1d1fe5 100644
--- a/doc/build-windows.md
+++ b/doc/build-windows.md
@@ -110,7 +110,7 @@ This means you cannot use a directory that is located directly on the host Windo
 
 Acquire the source in the usual way:
 
-    git clone https://github.com/Bitcoin-ABC/bitcoin-abc.git
+    git clone https://github.com/LogosFoundation/lotusd.git
 
 Once the source code is ready the build steps are below:
 
diff --git a/doc/files.md b/doc/files.md
index 928ff71d40..8723d4dfbe 100644
--- a/doc/files.md
+++ b/doc/files.md
@@ -102,7 +102,7 @@ Path           | Description | Repository notes
 `blkindex.dat` | Blockchain index BDB database; replaced by {`chainstate/`, `blocks/index/`, `blocks/revNNNNN.dat`<sup>[\[2\]](#note2)</sup>} in 0.8.0 | [PR #1677](https://github.com/bitcoin/bitcoin/pull/1677)
 `blk000?.dat`  | Block data (custom format, 2 GiB per file); replaced by `blocks/blkNNNNN.dat`<sup>[\[2\]](#note2)</sup> in 0.8.0 | [PR #1677](https://github.com/bitcoin/bitcoin/pull/1677)
 `addr.dat`     | Peer IP address BDB database; replaced by `peers.dat` in [0.7.0](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.7.0.md) | [PR #1198](https://github.com/bitcoin/bitcoin/pull/1198), [`928d3a01`](https://github.com/bitcoin/bitcoin/commit/928d3a011cc66c7f907c4d053f674ea77dc611cc)
-`onion_private_key` | Cached Tor onion service private key for `-listenonion` option. Was used for Tor v2 services; replaced by `onion_v3_private_key` in [0.22.14](https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/doc/release-notes/release-notes-0.22.14.md) | [D9201](https://reviews.bitcoinabc.org/D9201)
+`onion_private_key` | Cached Tor onion service private key for `-listenonion` option. Was used for Tor v2 services; replaced by `onion_v3_private_key` in [0.22.14](https://github.com/LogosFoundation/lotusd/blob/master/doc/release-notes/release-notes-0.22.14.md) | [D9201](https://reviews.bitcoinabc.org/D9201)
 
 ## Notes
 
diff --git a/doc/release-notes/release-notes-0.21.8.md b/doc/release-notes/release-notes-0.21.8.md
index c26ce0ada9..f334ab18c6 100644
--- a/doc/release-notes/release-notes-0.21.8.md
+++ b/doc/release-notes/release-notes-0.21.8.md
@@ -94,7 +94,7 @@ The `importmulti` RPC now supports importing of addresses from descriptors.
 A `desc` parameter can be provided instead of the `scriptPubKey` in a request,
 as well as an optional range for ranged descriptors to specify the start and
 end of the range to import. More information about descriptors can be found
-[here](https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/doc/descriptors.md).
+[here](https://github.com/LogosFoundation/lotusd/blob/master/doc/descriptors.md).
 
 New RPC methods
 ---------------
diff --git a/doc/release-process.md b/doc/release-process.md
index 100cea10c0..ad57450f8d 100644
--- a/doc/release-process.md
+++ b/doc/release-process.md
@@ -40,7 +40,7 @@ Bitcoin ABC Release Process
 
 9. Upload GUIX Builds to [bitcoinabc.org](https://download.bitcoinabc.org/)
 
-10. Create a [GitHub release](https://github.com/Bitcoin-ABC/bitcoin-abc/releases):
+10. Create a [GitHub release](https://github.com/LogosFoundation/lotusd/releases):
     `contrib/release/github-release.sh -a <path to release binaries> -t <release tag> -o <file containing your Github OAuth token>`
 
 11. Create [Ubuntu PPA packages](https://launchpad.net/~bitcoin-abc/+archive/ubuntu/ppa):
diff --git a/src/clientversion.cpp b/src/clientversion.cpp
index b6b3c241f8..d805cf1cd2 100644
--- a/src/clientversion.cpp
+++ b/src/clientversion.cpp
@@ -92,8 +92,8 @@ std::string CopyrightHolders(const std::string &strPrefix) {
 
 std::string LicenseInfo() {
     const std::string URL_SOURCE_CODE =
-        "<https://github.com/Bitcoin-ABC/bitcoin-abc>";
-    const std::string URL_WEBSITE = "<https://www.bitcoinabc.org>";
+        "<https://github.com/LogosFoundation/lotusd>";
+    const std::string URL_WEBSITE = "<https://www.givelotus.org>";
 
     return CopyrightHolders(strprintf(_("Copyright (C) %i-%i").translated, 2009,
                                       COPYRIGHT_YEAR) +
